collectors/iscsi.py

```diff
@@ -0,0 +1,247 @@
+#!/usr/bin/env python2
+
+# requires python-rtslib_fb for LIO interaction
```

Comment:
@zmc Agree - I think packaging needs more thought. We shouldn't just add the dependency, since this would install rtslib and targetcli to non-iscsi ceph nodes. However, every RHEL iscsi gateway will have the ceph-iscsi rpms installed, which pulls in rtslib_fb...I think this packaging discussion is bigger than this commit - maybe raise an issue on it for us to discuss?